physionet-django/events/forms.py

```diff
@@ -60,3 +60,17 @@ class EventApplicationResponseForm(forms.ModelForm):
     class Meta:
         model = EventApplication
         fields = ('status', 'comment_to_applicant')
+
+
+class EventAgreementForm(forms.ModelForm):
+    class Meta:
+        model = EventAgreement
+        fields = (
+            'name',
+            'version',
```

Comment:
We need to verify the given name/version doesn't already exist, otherwise the server will 500.

Can borrow some code from here:

https://github.com/MIT-LCP/physionet-build/blob/dev/physionet-django/project/forms.py#L387-L392

Reply:
In case of projects, we had loaded all the previous project for that slug  on the variable `previous_projects` 

But for Event Agreement, that might not be the best idea to follow that implementation because our `unique_together` criteria has title, version, creator as unique. So we need to check for the title+version+creator or make different validation function for each.(the suggested method worked for the published project because we only cared about not having duplicate version inside same projects)